first_lessons_dushanbe/array_sorting_binary_search.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+def list_sort(a):
```

Comment:
Why you add this method instead of using implementation from other file by referencing with `from file import func_name` ?